tools/install-deps.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
-# This scripts require CASSANDRA_VERSION, CASSANDRA_STRATIO_LUCENE_VERSION
-# to be set.
+# This scripts require CASSANDRA_VERSION, CASSANDRA_STRATIO_LUCENE_VERSION,
+# ES_VERSION to be set.
```

Comment:
What about using `ELASTICSEARCH_VERSION` for consistency?

Reply:
See comment above. I'm using `ES` prefix for consistency with testing.elasticsearch.